### ISSUE TITLE
Implement GCD and LCM in recursion

### ISSUE THREAD

**Issue by ephremdeme** [ephremdeme/data-structure-and-algorithms#106]
Implement GCD and LCM of two integers using any langauge

**Comment by dev-abir** [comment]
I can do this

**Comment by shaswatsingh19** [comment]
I would like to work on it in python

**Comment by neelansh15** [comment]
I can do it in Javascript

**Comment by ephremdeme** [comment]
On whatever language you choose you can implement it.

**Comment by neelansh15** [comment]
Shall I create a folder `recursion/GCD_and_LCM`?

**Comment by dev-abir** [comment]
yes, where we will store them?

**Comment by neelansh15** [comment]
@dev-abir Yeah. That is my question to the owner of the repo. I am implementing it in `recursion/GCD_and_LCM.js`. You can do it in your language

**Comment by dev-abir** [comment]
I will wait for the owner, before PRing

**Comment by dev-abir** [comment]
okay I will add this in recursion dir

**Comment by ephremdeme** [comment]
Hi, I have made directory restructure for mathematics related algorithms please check it out before PR. Thank you.

**Comment by neelansh15** [comment]
@ephremdeme  Hi. I've made the directory changes to PR #109. Please review it

**Comment by dev-abir** [comment]
@ephremdeme I have made necessary changes, please review it.

**Comment by Nivedita-coder** [comment]
Can I do this in C and python??

**Comment by himanshi2608** [comment]
Added code for gcd and lcm in cpp

**Comment by Pal-Sandeep** [comment]
I can do this in C and Python
